aidlc-inputs/00-business-brief.md

```
@@ -0,0 +1,361 @@
+
+**なぜ今サボれるのか、どこまでならサボっても大丈夫なのか**を具体的に説明するAI。
+
+---
+
+## 3. MVPでやること
+
+MVPでは、ランキングやSNS機能までは作らない。
+
+まずは、以下の2つに集中する。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)